reference/configuration/tidb-server/configuration-file.md

```diff
@@ -16,18 +16,28 @@ The TiDB configuration file supports more options than command-line parameters.
 - Default value: `true`
 - It is recommended to set it to `false` if you need to create a large number of tables.
 
-### `oom-action`
-
-- Specifies the operation when out-of-memory occurs in TiDB.
-- Default value: `log`
-- The valid options are `log` and `cancel`. `log` only prints the log without actual processing. `cancel` cancels the operation and outputs the log.
-
 ### `mem-quota-query`
 
 - The maximum memory available for a single SQL statement.
 - Default value: `34359738368`
 - Requests that require more memory than this value are handled based on the behavior defined by `oom-action`.
 
+### `oom-use-tmp-storage`
+
++ Controls whether to enable the temporary storage for some operators when a single SQL statement exceeds the memory quota specified by `mem-quota-query`.
++ Default value:  `true`
+
+### `tmp-storage-path`
+
++ Specifies the temporary storage path for some operators when a single SQL statement exceeds the memory quota specified by `mem-quota-query`.
++ Default value: `<TMPDIR>/tidb/tmp-storage`
```

Comment:
we should mention that this config only takes effect when `oom-use-tmp-storage` is set to `true`.